### ISSUE TITLE
Make snow toggleable please

### ISSUE THREAD

**Issue by nfc2005** [ThirteenAG/GTAIV.EFLC.FusionFix#700]
### What should happen

Not everyone lives in the Northern Hemisphere and not everyone gives Christmas the same meaning. This should be just like festive lights in 2DFX, an option and not forced upon the player. I'm aware of the phone cheat code to disable snow but come on.

### Checklist

- [X] Check the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already implemented.
- [X] [Search for other requests](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same feature.

**Comment by MatrakMatrix** [comment]
I didn't know this is supposed to be snow, thought there's something wrong with the game. Spent over an hour reinstalling, removing mods etc. only to end up here. Ridiculous.

**Comment by Ash-735** [comment]
We didn't think people would get so upset about this happening one day a year, so yes maybe Snow will be removed in the next update.

**Comment by nfc2005** [comment]
Not necessarily removed. It's nice that some people like it. And it makes sense that it can snow in Liberty City since it's based on New York. I just believe it should be an optional thing, hence why I ask about the possibility to toggle it. Not remove it.

**Comment by Jabarda** [comment]
The problem is that it doesn't look like snow, but rather resembles broken textures. Therefore, instead of enjoying Easter egg, people are looking for ways to return to how it was

**Comment by IVorza** [comment]
Yeah don’t remove it. Maybe make it an option in the ini file (for the specific day toggle). Like something from the likes of “Holiday Surprise” or “New Year Celebration” or something like that. Idk, just making a suggestion. And yes, I’m aware that this is togglable via 766-555-0100, but I like the surprise effect it has upon being toggled automatically upon reaching the 30th.

**Comment by Ash-735** [comment]
> The problem is that it doesn't look like snow, but rather resembles broken textures. Therefore, instead of enjoying Easter egg, people are looking for ways to return to how it was

It's handled exactly like how Rockstar do it with GTAO.

**Comment by ColesMan** [comment]
How exactly does this feature belong in an essential fix mod like this? The game is basically unplayable on systems nowadays without this mod (and others). Without mentioning this anywhere in the readme or ini file you're guaranteeing when new players are going to download this, they could see this, and at best mistakenly think it's an intended effect by Rockstar, or at worst think its a bug and try to 'fix' it, which is now what is happening. I get that it's a pretty cool effect but it really has no place being anything besides an entirely optional and voluntary addon.

**Comment by Othreute** [comment]
Not only the "snow" itself looks ugly, I have experienced light sources flickering during cutscenes (extremely distracting, sometimes entire screen blinked black for me frequently), and have been wondering what was causing it, even updated my drivers etc... Turning off this "feature" with 766-555-0100 has fixed everything.

Stupendously bad idea and bad implementation too. This does not belong in a fix mod, period.

**Comment by ZT-1** [comment]
The snow is pretty cool honestly and I don’t think it should be removed completely. But can see why some people don’t want it forced even for one day. Maybe some sort of seasonal events toggle in the ini defaulted to disabled would satisfy everyone.

**Comment by YuWongXX** [comment]
It is totally annoying.

It looks like broken textures instead of beautiful snow in RDR2.
This should not be included in a fix mod.
This is unnatural in the story of GTA4.

**Comment by Terraformer9x** [comment]
Remove the snow entirely, it doesn't belong in a mod like this. It actually made me think that something was wrong with the textures in my game.

Make it a separate mod if anything but please don't include out of the blue stuff like this.

**Comment by sTc2201** [comment]
Closing this, no need for more echoes.

A toggle will be implemented soon.

**Comment by sTc2201** [comment]
Fixed with commit b9313ee
